README.md

```diff
@@ -105,6 +105,7 @@ These simulation-based verification tools are available for Chisel:
 - [**Wiki**](https://github.com/freechipsproject/chisel3/wiki), which contains various feature-specific tutorials and frequently-asked questions.
 - [**ScalaDoc**](https://www.chisel-lang.org/api/latest/chisel3/index.html), a listing, description, and examples of the functionality exposed by Chisel
 - [**Gitter**](https://gitter.im/freechipsproject/chisel3), where you can ask questions or discuss anything Chisel
+- [**Style Guide**](https://github.com/ccelio/chisel-style-guide), A basic guide to chisel style conventions
```

Comment:
Is this meant to be an official style guide? Should it be prefixed?

This might be nitpicking; the style guide recommends looking at rocket as an example for beginners to read....

Reply:
Baby steps.

I think further discussion and work here would be well spent. I wonder if we should fork his repo, but his is the top hit if one search chisel-style guide.

I had planned to variable naming section. That's a simple discussion I'm sure.